napalm_yang/parsers/text.py

```diff
@@ -30,6 +31,13 @@ def _parse_list_block(cls, mapping):
                 else:
                     key = extra_vars.pop("key")
 
+                if post_process_filter:
+                    from napalm_yang.helpers import template
```

Comment:
Can't :( I'll get a circular import in that case, because of:

https://github.com/napalm-automation/napalm-yang/blob/dbarrosop/include/napalm_yang/helpers.py#L3

I can split `helpers.py` into two modules - one with functions dealing with YANG files/parsers (maybe keep them in `helpers.py`) and another that resolves/renders rules (move `_resolve_rule()`, `resolve_rule()´ and `template()´ into `rules.py`). Thoughts?